Changelog.md

```diff
@@ -1,5 +1,10 @@
 # Spectrometer Changelog
 
+## Unreleased
+
+- Add new hidden subcommand `experimental-vsi-register-binary-custom-dependency`.
+  This subcommand is intended to be used with direct support from FOSSA engineering when working with specific projects.
```

Comment:
Not 100% sure on this, but I think that if we don't want people using things, we should probably leave them out of the changelog.

Reply:
Fair enough, it just seemed odd to have a release with "nothing" in it (if I end up doing a release with just my changes). Happy to go either way.

Reply:
Yeah, I'm really not 100% on this.  Let's leave it in.  Also make sure to add the PR numbers to the changelog.